apps/modernization-ui/src/apps/search/event/components/LabReportSearch/LabReportGeneralFields.tsx

```diff
@@ -387,49 +387,20 @@ export const LabReportGeneralFields = ({ form }: LabReportGeneralFieldProps) =>
                     }}
                 />
             </Label>
-
-            <SearchCriteriaContext.Consumer>
-                {({ searchCriteria }) => (
-                    <>
-                        <Controller
-                            control={form.control}
-                            name="createdBy"
-                            render={({ field: { onChange, value, name } }) => (
-                                <AutoCompleteInput
-                                    name={name}
-                                    value={value}
-                                    onChange={onChange}
-                                    label="Event created by user"
-                                    options={searchCriteria.userResults.map((user) => {
-                                        return {
-                                            label: `${user.userLastNm}, ${user.userFirstNm}`,
-                                            value: user.nedssEntryId
-                                        };
-                                    })}
-                                />
-                            )}
-                        />
-                        <Controller
-                            control={form.control}
-                            name="lastUpdatedBy"
-                            render={({ field: { onChange, value, name } }) => (
-                                <AutoCompleteInput
-                                    name={name}
-                                    value={value as string | undefined}
-                                    onChange={onChange}
-                                    label="Event updated by user"
-                                    options={searchCriteria.userResults.map((user) => {
-                                        return {
-                                            label: `${user.userLastNm}, ${user.userFirstNm}`,
-                                            value: user.nedssEntryId
-                                        };
-                                    })}
-                                />
-                            )}
-                        />
-                    </>
+            <Controller
+                control={form.control}
+                name="createdBy"
+                render={({ field: { onChange } }) => (
+                    <UserAutocomplete id="createdBy" label="Event created by user" onChange={onChange} />
                 )}
-            </SearchCriteriaContext.Consumer>
+            />
+            <Controller
+                control={form.control}
+                name="lastUpdatedBy"
+                render={({ field: { onChange } }) => (
+                    <UserAutocomplete id="lastUpdatedBy" onChange={onChange} label="Event updated by user" />
```

Comment:
react hook form makes the `name` passed to the `Controller` available in the render function so you don't have to repeat it for the label.

It allows you to do something like 

```
 render={({ field: { name, onChange } }) => (
                    <UserAutocomplete id={name} onChange={onChange} label="Event updated by user" />
```